src/rpcblockchain.cpp

```diff
@@ -395,12 +395,15 @@ UniValue getblock(const UniValue& params, bool fHelp)
     if (fHelp || params.size() < 1 || params.size() > 2)
         throw runtime_error(
             "getblock \"hash|height\" ( verbose )\n"
-            "\nIf verbose is false, returns a string that is serialized, hex-encoded data for block 'hash|height'.\n"
-            "If verbose is true, returns an Object with information about block <hash|height>.\n"
+            "\nIf verbosity is 0, returns a string that is serialized, hex-encoded data for block 'hash'.\n"
```

Comment:
In several places the documentation says "block 'hash'" or something similar, but this is actually hash or height. Is this something we care to update?

Reply:
I tidied up the help which we inherited from upstream.